### PR TITLE
About site

### PR DESCRIPTION
**Pull request by joern-vh**
Solves #88.

### CODE REVIEW

File: client/components/About.js [hunk 1]
```diff
@@ -10,14 +10,19 @@ class About extends React.Component {
       <div className='about'>
         <Header />
         <div className='container-fluid animated fadeIn'>
+          <div className='row text-center'>
+            <h3>About the Project</h3>
+          </div>
           <img className='group-photo' src={groupPhoto} />
```
Comment (kordianbruck) [inline client/components/About.js:16]: Please move the image as suggested by @goldbergtatyana here https://github.com/MusicConnectionMachine/VisualizationG6/issues/88#issuecomment-296456810

**Review by kordianbruck — CHANGES_REQUESTED** [review]

File: client/components/About.js (hunk 1 above)
Comment (joern-vh) [reply client/components/About.js:16]: Ah sorry, I misunderstood. Thought that just the title should be moved.

File: client/components/About.js [hunk 2]
```diff
@@ -10,14 +10,19 @@ class About extends React.Component {
...
-                We summarize the data available about composers, music works and musicians as a set of connections - relationships between composers, relationships between composers and music works, between music works and musicians, composers and locations, dates and much more.
+                The Music Connection Machine has been developed within the JavaScript Course at the Technical University of Munich.<br /><br />
+                Information about classical music is scattered all over the internet in the form of scholarly articles, news stories, blogs, wikis, forums and many other venues.
+                Our goal is to bring this knowledge into one place and make it easily accessible.
+                We summarize the information about composers, musicians and music works as a set of connections -
+                what were the musicians saying about each other and the music works, and what anybody else has written about them online.
+                Moreover, whenever we find temporal  or location-based  information, we can present this  information in geographic and historic context.
```
Comment (chaoran-chen) [inline client/components/About.js:25]: Maybe, it's better to write "we present this information ..." instead of "we can present ..."?

**Comment by felixschorer** [comment]
Could you also fix #118 while you're at it? Please? :D

**Comment by joern-vh** [comment]
@felixschorer Sure ;)

**Comment by joern-vh** [comment]
Closed by #120.

**Pull request closed without merging**